### ISSUE TITLE
[Bug]: Mod is not working on modpack server

### ISSUE THREAD

**Issue by YejiYeah** [The-Fireplace/Simply-Caterpillar#18]
### Mod Version

8.0.1

### What mod loader(s) are you seeing the problem on?

Fabric

### Mod Loader Version

0.14.22

### Problem Description

 (Fabric - 1.20.1) The server doesn't load when the mod is installed in the server. Removing the mod fixed the issue.

### Steps To Reproduce

1. Add the mod in the mod's folder in the server
2. Run the server
3. The server doesn't start
4. There is a line related to the mod in the latest.log 

### Crash Report or Error Logs

```shell
https://gist.github.com/SaraXianLin/8e09d08c452d17e6353c71efa62806a3
```

**Comment by Daniel-Mendes** [comment]
You're right the mod doesn't work on servers currently.

**Comment by Daniel-Mendes** [comment]
Fixed with release of 8.0.2
